# Different gait types

float32 foot_clearance
float32 base_height
float32[] timing_parameters
float32 max_forward_speed
float32 recommended_forward_speed
time timestamp
